contracts/meta-oracles/LinearizedPriceDataSource.sol

```diff
@@ -21,7 +21,8 @@ import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 import { IMedian } from "../external/DappHub/interfaces/IMedian.sol";
```

Comment:
This might be in a future PR, but I'm assuming we want to use an interface that returns uint256 vs `IMedian` for the version we deploy?

Reply:
Yeah future PR